bundles/org.openhab.binding.metofficedatahub/README.md

```diff
@@ -0,0 +1,463 @@
+
+This binding supports the follow thing types:
+
+| Thing             | Thing Type | Thing Type UID  | Discovery | Description                                                                                 |
+|-------------------|------------|-----------------|-----------|---------------------------------------------------------------------------------------------|
+| Bridge            | Bridge     | bridge          | Manual    | A single connection to the Met Office DataHub API with daily poll limiting for the Site API |
+| Site Specific API | Thing      | siteSpecificApi | Manual    | Provides the hourly and daily forecast data for a give location (site)                      |
+
+## Thing Configuration
+
+### Bridge configuration parameters
+
+| Name                       | Type   | Description                                                                                                                     | Recommended Values |
+|----------------------------|--------|---------------------------------------------------------------------------------------------------------------------------------|--------------------|
+| siteSpecificRateDailyLimit | Number | For the runtime of the system, this is the limit of how many polls for updates are allowed for updates for the SiteSpecific API | 200                |
+| siteSpecificClientId       | String | The number of hours between polling for each sites daily data                                                                   | 3                  |
+| siteSpecificClientSecret   | String | The poll interval (seconds) for air filters / humidifiers                                                                       | 60                 |
+
+**NOTE:** siteSpecificRateDailyLimit: This **should** prevent any more poll's for the rest of the day to the SiteSpecific API, once this limit is reached as a failsafe against a bad configuration,
+if you don't reboot / delete and re-add the bridge.
+
+### Site Specific API configuration parameters
+
+| Name                               | Type   | Description                                                    | Recommended Values |
+|------------------------------------|--------|----------------------------------------------------------------|--------------------|
+| siteSpecificHourlyForecastPollRate | Number | The number of hours between polling for each sites hourly data | 1                  |
+| siteSpecificDailyForecastPollRate  | Number | The number of hours between polling for each sites daily data  | 3                  |
+| location                           | String | The lat/long of the site e.g. "51.5072,0.1276"                 |                    |
+
+## Channels
+
+This binding uses channel groups. The channels under "Forecast for the current hour" will be mirrored for future hours
+forecasts.
+
+The channel naming follows the following format:
```

Comment:
I think it is better to move and improve this example bwlow the channel table.
It owuld also be easier to read if all declarations have code markup.

Reply:
Thank - yes that flows better, I have adjusted.